src/ontology/mondo-edit.obo

```diff
@@ -205817,14 +205817,16 @@ property_value: confidence "6.142857142857142" xsd:double
 
 [Term]
 id: MONDO:0010582
-name: diabetes insipidus, neurohypophyseal type
+name: diabetes insipidus, neurohypophyseal type, X-linked inheritance
 synonym: "diabetes insipidus, neurohypophyseal type" EXACT [OMIM:304900]
 xref: OMIM:304900 {source="MONDO:equivalentTo"}
 xref: Orphanet:178029 {source="MONDO:subClassOf", source="OMIM:304900"}
 xref: Orphanet:30925 {source="MONDO:subClassOf", source="OMIM:304900"}
 xref: UMLS:C0687720 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:304900"}
 xref: UMLS:CN074293 {source="NCBI:mim2gene_medline", source="MONDO:equivalentTo"}
-is_a: MONDO:0007450 {source="MONDOLEX:0010582", source="ORDO:30925/btnt"} ! neurohypophyseal diabetes insipidus
+is_a: MONDO:0004782 {source="OMIM:304900"} ! diabetes insipidus
+relationship: excluded_subClassOf MONDO:0007450 {source="MONDOLEX:0010582", source="ORDO:30925/btnt"} ! neurohypophyseal diabetes insipidus
```

Comment:
This doesn't make sense terminologically. if we have disease FOO, and FOO-type-X, then FOO-type-X should be a subclass of FOO,  **OR** we should relabeled FOO to be something less misleadingly generic

Consult the DPs, particularly X-linked inheritance